app/src/main/java/com/codepath/apps/critterfinder/activities/PetDetailsActivity.java

```diff
@@ -37,6 +38,8 @@ protected void onCreate(Bundle savedInstanceState) {
 
         Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
         setSupportActionBar(toolbar);
+        // add a back button
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
```

Comment:
nice job adding the back button! if I click on the back button does it dismiss the pet details activity? I wonder if we have to do something else to notice that the back button was clicked and to dismiss the activity.